apps/src/templates/sectionProgress/sectionProgressRedux.js

```diff
@@ -84,6 +84,13 @@ export const studentLevelProgressPropType = PropTypes.objectOf(
   PropTypes.objectOf(PropTypes.number)
 );
 
+export const jumpToLessonDetails = (lessonOfInterest) => {
```

Comment:
nit: in general we've kept these actions grouped together towards the top of the file at line ~19 so they are organized/easy to find.